ci/build-subprojects.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -ev
+
+case "$SUBPROJECT" in "rosette")
+    cd rosette
+    nix-build
+
+     ./run.sh rbl/rosette/tests/simple_add.rbl
+     ;;
```

Comment:
Nitpicks: formatting is a bit wonky here.

You have different indentation levels in this file (I see some 1s and 5s).  You probably should just stick to 4.  

Each case like `"core")` should either be at 0, 2, or 4.  (I tend to put them at 0).

It's also customary when writing case statements to but each case on its own line, including the starting case. 
 So the initial lines of the case statement should be something like:

```sh
case "$SUBPROJECT" in 

"rosette")
    cd rosette
```

Reply:
Not nitpicks. My bad. I think it was just a little too late. I shouldn't have missed those. I fixed it up.

Reply:
I don't see this fixed. Did you forget to push?